### PR TITLE
Added encoder support, fixed keymap between via and Cory Dora mapping QK_BOOT to flash

### PR DESCRIPTION
**Pull request by layogtima**
## Description

Changes,
- Rotary encoder for Cory Dora should now as expected (added mappings and reasonable defaults)
- Via / QMK mappings now match for the Cory Dora
- QK_BOOT to flash is now enabled to Key 9, profile #4 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [✔] Keyboard (addition or update)
- [✔] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [✔] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [✔] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [✔] My change requires a change to the documentation.
- [✔] I have updated the documentation accordingly.
- [✔] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [✔] I have added tests to cover my changes.
- [✔] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by layogtima** [comment]
@balub see if this works!

File: keyboards/handwired/corydora/via.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
         "keymap": [
             ["0,0", "0,1", "0,2"],
             ["1,0", "1,1", "1,2"],
-            ["2,0", "2,1", "2,2"]
+            ["2,0", "2,1", "2,2", "3,0\n\n\n\n\n\n\n\n\ne0"]
```
Comment (balub) [inline keyboards/handwired/corydora/via.json:13]:
```suggestion
            ["2,0", "2,1", "2,2", "3,0"]
```

**Review by balub — CHANGES_REQUESTED** [review]

**Comment by layogtima** [comment]
Removed redundant config repeats. Fixed the via.json file issue; rows need to be 4!

**Review by balub — APPROVED** [review]
LGTM 🚀

**Pull request merged**